ksqldb-rest-app/src/main/java/io/confluent/ksql/api/auth/AuthenticationPluginHandler.java

```diff
@@ -107,6 +108,19 @@ public void handle(final RoutingContext routingContext) {
       this.principal = new DefaultKsqlPrincipal(principal);
     }
 
+    @Override
+    public JsonObject attributes() {
```

Comment:
What are the new methods used for?

Reply:
we bastardize the vertx authentication  😅   these methods are to help support the json ser/de of the principals, but we don't use that

Reply:
![image](https://user-images.githubusercontent.com/3172405/162278903-0f2c50c6-a70d-4e35-8908-30d1661f3499.png)

We don't use that method at all because we have an SPI for creating principals and not a json API